CONTRIBUTING.md

```diff
@@ -9,11 +9,15 @@ and ask that all contributors do the same.
 
 ### Git Protocol
 
-To contribute to this project, people internal to 18F can create a branch and submit a pull request. If you are external to 18F, you can fork the repository and submit a pull request that way.
+To contribute to this project, people internal to 18F can create a branch and 
+submit a pull request. If you are external to 18F, you can fork the repository 
+and submit a pull request that way.
 
-We are minimizing commits on the `master` branch by squashing commits and force pushing them to `master`. 
+We are minimizing commits on the `master` branch by squashing commits and force 
```

Comment:
this is inaccurate and I changed it in https://github.com/18F/dolores-landingham-bot/pull/56